genai-perf/genai_perf/parser.py

```diff
@@ -300,6 +300,16 @@ def _check_goodput_args(args):
     return args
 
 
+def _check_session_args(args):
+
+    if args.num_sessions:
+        if args.session_concurrency is None:
```

Comment:
Why not default to 1 for `session_concurrency`?
Trying to fail early to notify the user?

Reply:
Great question! I want to avoid default logic right now, because we might end up requiring this anyway after Brian's config PR goes through. If we create a mutually-exclusive load management group, then one of them will be required. So I'm trying to avoid instituting something that will break in the next couple of weeks.

Reply:
Got it. That works for me.